src/faq.md

```diff
@@ -446,6 +446,50 @@ The reference runner and several other CWL implementations support running
 those Docker format containers using the Singularity engine. Directly
 specifying a Singularity format container is not part of the CWL standards.
 
+## How outputBinding.glob works
+The `outputBinding` section describes how to set the value of each output parameter.  The `glob` field is used to return all files/pathnames that match a specific pattern. The `glob` field specifies the file and the output directory in which the file should be stored. The value of the glob field is the `output directory`.  
+If the value of glob is a relative path pattern (one that does not start with a slash '/'), it is resolved relative to the output directory. If the glob value is an absolute path pattern (it starts with a slash '/'), it must refer to a path within the output directory.
+Glob's pattern rules abide to standard Unix path expansion rules. According to benchmarks, it is also expected to be faster than other methods for matching pathnames in directories.
+Wildcard characters can be used in the filenames when searching for files. The wildcard characters can either be an asterisk `*`, a question mark matching pathnames in directories. `?` or a range `[]`
+
+In the example below, the glob field is used to specify the file(`hello.txt`) in which the output should be stored.
+
+```cwl
+outputs:
+  example_out:
+    type: File
+    outputBinding:
+      glob: hello.txt
+```  
+
+The glob field in the above output section can also be written with the `*` wildcard character as
+
+```cwl
+outputs:
+  example_out:
+    type: File
+    outputBinding:
+      glob: '*.txt'
+```
+
+Glob can also be used to return all outputs from a CommandLine Tool. For example
+
+```
+  outputs:
+  type: Directory
+  outputBinding: { glob: . }
+```
+
+OR
+
+```
+  outputs:
+  type:
+    type: array
+    items: [Directory, File]
+  outputBinding: {glob: "*"}
+```
+
```

Comment:
Good to add an example using input values to define the glob.

Reply:
outputs:
  type:
    type: array
    items: [Directory, File]
 this seems odd to me.  do you have a reference for this -- because I might be wrong but this looks like an array but the array only has 1 directory and 1 File ?